src/saveload/afterload.cpp

```diff
@@ -2203,7 +2202,8 @@ bool AfterLoadGame()
 			}
 
 			if (remove) {
-				DeleteAnimatedTile(*tile);
+				tile = _animated_tiles.erase(tile);
+				MarkTileDirtyByTile(*tile);
```

Comment:
Why do you even need to mark dirty tiles in afterlolad?

Reply:
I copied over from DeleteAnimatedTile. But if it doesn't need that, I can remove this line.